statvar_imports/us_education/florida_education/pvmap_enrollment.csv

```diff
@@ -0,0 +1,14 @@
+key,p1,v1,p2,v2,p3,v3,p4,v4,,
+Sum of Headcount (#),value,{Number},populationType,Student,measuredProperty,count,statType,measuredValue,observationAbout,dcs:geoId/12
+Fall Term,observationDate,{Data},,,,,,,,
+#Graduate,enrollmentLevel,PostGraduate,,,,,,,,
+Undergraduate,enrollmentLevel,Graduate,,,enrollmentStatus,Enrolled,,,,
+"Doctorate: Professional (ex. M.D., J.D.)",enrollmentLevel,DoctorateDegree,doctorateDegreeMajor,ProfessionalExcludingMDAndJD,enrollmentStatus,Enrolled,,,,
+Doctorate: Research (ex. Ph.D.),enrollmentLevel,DoctorateDegree,doctorateDegreeMajor,ResearchExcludingPhD,enrollmentStatus,Enrolled,,,,
+Masters,enrollmentLevel,MastersDegree,,,enrollmentStatus,Enrolled,,,,
+FCS AA Transfer,schoolEnrollment,TransferredFromFCSAA,,,,,,,,
+#First-Time-in-College (FTIC),schoolEnrollment,FirstTimeInCollege,,,,,,,,
```

Comment:
schoolEnrollment does not seem to fit the context. Can we use the studentType with new Enum value.